src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator start working!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stop working!");
```

Comment:
Consider making the stop message more informative as well. For instance: 'Excavator has completed its work and is stopping.' This aligns with the checklist requirement for informative messages.